benchmarks/add.py

```diff
@@ -0,0 +1,30 @@
+import shutil
+
+from dvc.main import main
+
+from benchmarks.base import BaseBench, init_dvc, random_data_dir
+
+
+class Add_100_1M_copy(BaseBench):
+    def setup(self):
+        super().setup()
+        self.repo = init_dvc(self.test_directory.name)
+        dataset_path = random_data_dir(100, 1024 * 1024)
+        shutil.copytree(dataset_path, "data")
+
+    def time_add(self):
+        assert main(["add", "data"]) == 0
```

Comment:
As we've discussed, `main` will launch stuff like updater/analytics workers, which might affect the results. We need to check that.

Reply:
Turning analytics off during initialization.